src/integration/java/com/salesforce/dynamodbv2/mt/admin/AmazonDynamoDbAdminUtilsIt.java

```diff
@@ -0,0 +1,122 @@
+package com.salesforce.dynamodbv2.mt.admin;
```

Comment:
Is there a benefit to putting this in src/integration/java instead of src/test/java?

Reply:
I was following the convention of our other projects.  I think it's helpful to have them separate, it's easier to differentiate the two when first looking at the project.